MANIFEST.in

```diff
@@ -4,3 +4,5 @@ include requirements.txt
 
 recursive-exclude * __pycache__
 recursive-exclude * *.py[co]
+
+graft iterseg/data
```

Comment:
try changing this to:

```
graft src/iterseg/data
```

?

Reply:
ah, that includes all the .zarr files which I don't think we want to ship. Maybe:

```suggestion
graft src/iterseg/data/*.pt
```

Reply:
... still bringing in the zarrs. wth.

Reply:
maybe move the zarrs (and csv) to a different dir, say, `_local_data/`?

Reply:
ok, sorry for the wild goose chase — this seems to work:

```suggestion
recursive-include data *.pt
```

Still building but I see this line in the logs:

```
creating iterseg-0.1.0/data
```

which is encouraging!

Reply:
Weird... The *source* distribution has it. But not the wheel. 🤔